inclusive-design/idrc
# feat: minimal team page (resolves #55)

## Description

Adds minimal implementation of a team page. Individual team member pages are created but not linked from the team page and not styled yet.

TODO:

- [x] Add people to CMS for editing.

## Steps to test

1. Visit deploy preview.
2. Review _About / Team_ page.

**Expected behavior:** Matches designs.

## Additional information

Not applicable.

## Related issues

Resolves #55.
